libs/dyn/mapping_test.go

```diff
@@ -125,8 +125,8 @@ func TestMappingSetLoc(t *testing.T) {
 func TestMappingKeysValues(t *testing.T) {
 	// Configure mapping
 	var m dyn.Mapping
-	m.SetLoc("key1", []dyn.Location{}, dyn.V("foo"))
```

Comment:
There are other places in the code where we use []dyn.Location{}, should it be changed as well?

Reply:
Updated all.